playbook/app/pb_kits/playbook/pb_star_rating/_star_rating.tsx

```diff
@@ -2,40 +2,88 @@ import React from "react"
 import classnames from "classnames"
 
 import { buildAriaProps, buildDataProps } from "../utilities/props"
-
-import Icon from "../pb_icon/_icon"
+import Caption from '../pb_caption/_caption'
+import Body from '../pb_body/_body'
+import Title from '../pb_title/_title'
 
 type StarRatingProps = {
   aria?: {[key: string]: string},
   className?: string,
   data?: object,
+  dark?: boolean,
   fixedWidth?: boolean,
-  hideRating: boolean,
+  layoutOption?: "default" | "number" | "onestar",
   icon?: string,
   id?: string,
   rating: number,
+  denominator: number,
+  colorOption?: "yellow" | "primary" | "subtle" | "outline",
+  size?: "xs" | "sm" | "md" | "lg";
 };
 
 const StarRating = ({
   aria = {},
   className,
   data = {},
-  hideRating = false,
+  dark = false,
+  layoutOption = "default",
   id,
   rating = 0,
+  denominator = 5,
+  colorOption = "yellow",
+  size = "sm",
 }: StarRatingProps) => {
   const ariaProps = buildAriaProps(aria)
   const dataProps = buildDataProps(data)
   const css = classnames([
     "pb_star_rating_kit", className,
   ])
 
-  const starCount = () => (
-    [...Array(Math.floor(rating))]
+  const denominatorStyle = layoutOption === "onestar" ? 1 : denominator
+  const activeStars = Math.round(rating) > denominatorStyle ? denominatorStyle : Math.round(rating)
+  const emptyStars = denominatorStyle - Math.round(rating) < 0 ? 0 : denominatorStyle - Math.round(rating)
+  let svgSize
+
+  if (size === 'xs') {
+    svgSize = 14;
+  } else if (size === 'sm') {
+    svgSize = 16;
+  } else if (size === 'md') {
+    svgSize = 24;
+  } else if (size === 'lg') {
+    svgSize = 48;
+  } else {
+    svgSize = 16;
+  }
```

Comment:
There are ways you can setup your SVG to scale with CSS. Thats how all our icons work. These feel like they should be defined in SCSS since they are display values.

[Check out some of the description under custom icons](https://playbook.powerapp.cloud/kits/icon/react#icon-custom)

Reply:
I am using the icon kit now